docs/deployment/landing_page.rst

```diff
@@ -384,3 +384,27 @@ SecureDrop submissions. We control the servers that store your submissions,
 so no third party has direct access to the metadata or content of what you send us.
 
 Do not discuss leaking or whistleblowing, even with trusted contacts.
+
+How to test your *Landing Page* using Tor Browser
+-------------------------------------------------
+
+*Sources* may visit your *Landing Page* using Tor.
+
+Many websites are configured with security measures that only apply
+when Tor is in use. For example, Tor visitors may be requested to solve
+a CAPTCHA, may trigger warnings that are specific to some Tor exit nodes,
+or may be unable to load the page altogether because of
+Tor-specific DDoS protections.
+
+The effect of such measures cannot be tested without using Tor, and it is
+a very bad experience for a *source* if visiting a *Landing Page* doesn't work
+as expected. Because of that, it is **strongly recommended** that you test
```

Comment:
I would change this to active voice (it is recommended -> we recommend).

Reply:
Done.